### PR TITLE
Excluding .tt files from packages

### PR DESCRIPTION
**Pull request by vadym-vorotilin**

### CODE REVIEW

File: src/Vim.Math3D.csproj [hunk 1]
```diff
@@ -54,6 +54,8 @@
       <Pack>True</Pack>
       <PackagePath></PackagePath>
     </None>
+
+    <Content Remove="*.tt" />
```
Comment (mavimaec) [inline src/Vim.Math3D.csproj]: @Vadim-Vorotilin did you test whether the .tt file text generation continues to work with this `Remove` ? For example, does a change to Structs.tt still produce a corresponding update in Structs.cs?

**Review by cdiggins — APPROVED** [review]
Code looks good, Martin's test suggestion, is a good idea.

File: src/Vim.Math3D.csproj (hunk 1 above)
Comment (vadym-vorotilin) [reply src/Vim.Math3D.csproj]: Good question. I'll look into it. Thanks!

**Comment by vadym-vorotilin** [comment]
This method didn't work correctly. Fixed it on the CI/CD level.

**Pull request closed without merging**